README.md

```diff
@@ -100,4 +100,17 @@ You can also use tools like certmanager's [trust operator](https://cert-manager.
 ## Development
 `make build`
 
-`docker build -f package/Dockerfile . -t $MY_REPO:$MY_TAG`
\ No newline at end of file
+`docker build -f package/Dockerfile . -t $MY_REPO:$MY_TAG`
+
+## Release
+
+1. Update the [rancher-csp-adapter chart](https://github.com/rancher/charts/blob/dev-v2.6/packages/rancher-csp-adapter/package.yaml):
+    1. Clone this repo (not a fork).
+    2. Pull the `release/v2.6` branch.
+    3. Tag the branch with a new version. For example, to release version `1.1.2`, tag with `git tag v1.1.2` (make sure to include the `v`, it's important for the CI).
+    4. Push the new tag to the remote: `git push --tags`. Wait for the pipeline to release the new version before moving on to the next step.
+    5. In the charts repo, update the version of the adapter. See the [rancher/charts](https://github.com/rancher/charts/tree/dev-v2.6) repo for instructions on updating the [rancher-csp-adapter chart](https://github.com/rancher/charts/blob/dev-v2.6/packages/rancher-csp-adapter/package.yaml).
+2. Update the value of the `CATTLE_CSP_ADAPTER_MIN_VERSION` environment variable in [Rancher's Dockerfile](https://github.com/rancher/rancher/blob/release/v2.6/package/Dockerfile).
+3. Update the version compatibility matrix in the main Rancher docs.
+4. Check Kubernetes and Rancher version limits in this repo's `charts/Chart.yaml`.

```

Comment:
I think this step would go before step 1 since you need to check them before you can release the charts. We also might want to indicate what someone is looking for and when/why you would need to update this value - something like `change the supported kubernetes versions if you have added/removed support for a version in the current range. Change the rancher version only when making a new version of the csp-adapter.`